src/Identity.API/Program.cs

```diff
@@ -17,7 +17,7 @@
 
 builder.Services.AddIdentityServer(options =>
 {
-    options.IssuerUri = "null";
+    //options.IssuerUri = "null";
```

Comment:
Why this change?

If we are going to make it, should the line just be deleted?

Reply:
Having this value set caused the mobile client to fail authentication. I have no problem removing it though.